src/web/mjs/engine/Favorites.mjs

```diff
@@ -0,0 +1,66 @@
+export default class Favorites {
+
+    constructor(ipc) {
+        ipc.listen('on-connector-protocol-handler', this._onConnectorProtocolHandler.bind(this));
+        this._list = [];
+    }
+
+    async _loadPlugins(uri) {
+        try {
+            let response = await fetch(uri);
+            let data = await response.json();
+            let cc = data.filter(plugin => plugin.startsWith('hakuneko.mangas.'));
+            let connectorName = [];
+            cc.forEach(file => {
+                let name = file.substr(file.indexOf('.', 15)+1);
+                connectorName.push("hakuneko://cache/mjs/connectors/" + name + ".mjs");
+            });
+            return connectorName;
```

Comment:
feel like this could've been done better, specially since it needs to be integrated into the original `Connectors.mjs`
requesting feedback

Reply:
Move this method into the `Storage.mjs` engine and use the appropriate methods available or `this.fs` directly (see `saveMangaList` and `loadMangaList`)

Reply:
There is also a private method `_readDirectoryEntries(path)` which may come in handy